### PR TITLE
app-alternatives/bzip2!

### PR DESCRIPTION
**Pull request by mgorny**
The idea is to install `app-arch/bzip2` as `bzip2-reference` and control `bzip2`, `bunzip2` and `bzcat` symlinks between `bzip2-reference`, `lbzip2` and `pbzip2`. Now usr-merge friendly!

I haven't touched other fancy symlinks like `bzless` since they seem reference implementation-specific (also do we really want them?)

### CODE REVIEW

File: app-arch/bzip2/bzip2-1.0.8-r4.ebuild [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+# XXX: atm, libbz2.a is always PIC :(, so it is always built quickly
+#      (since we're building shared libs) ...
+
+EAPI=7
+
+VERIFY_SIG_OPENPGP_KEY_PATH="${BROOT}"/usr/share/openpgp-keys/bzip2.gpg
+inherit toolchain-funcs multilib-minimal usr-ldscript verify-sig
+
+DESCRIPTION="A high-quality data compressor used extensively by Gentoo Linux"
```
Comment (mgorny) [inline app-arch/bzip2/bzip2-1.0.8-r4.ebuild:12]: I suppose this is no longer up-to-date ;-).

**Comment by thesamesam** [comment]
>I haven't touched other fancy symlinks like bzless since they seem reference implementation-specific (also do we really want them?)

Nah, because e.g. `xzless` actually handles other formats too. So just leave that.

Can you do gzip next? ;)

**Comment by mgorny** [comment]
> Can you do gzip next? ;)

I was thinking about that. That's just pigz, right?

**Comment by thesamesam** [comment]
Yeah, just pigz.

**Pull request closed without merging**